packages/math/atoms.lua

```diff
@@ -0,0 +1,38 @@
+local atomType = {
+   ordinary = 0,
+   bigOperator = 1,
+   binaryOperator = 2,
+   relationalOperator = 3,
+   openingSymbol = 4,
+   closeSymbol = 5,
+   punctuationSymbol = 6,
+   inner = 7,
+   overSymbol = 8,
+   underSymbol = 9,
+   accentSymbol = 10,
+   radicalSymbol = 11,
+   vcenter = 12,
```

Comment:
Just for the record, keeping those unused atom types for now until more things are cleared up, but eventually some will be removed or reassigned.

Reply:
Cleaning performed